plot-api/src/commonMain/kotlin/org/jetbrains/letsPlot/LetsPlot.kt

```diff
@@ -6,14 +6,30 @@
 package org.jetbrains.letsPlot
 
 import org.jetbrains.letsPlot.frontend.NotebookFrontendContext
+import org.jetbrains.letsPlot.intern.Feature
+import org.jetbrains.letsPlot.intern.FeatureList
 import org.jetbrains.letsPlot.intern.OptionsMap
 import org.jetbrains.letsPlot.intern.settings.GlobalSettings
 import org.jetbrains.letsPlot.intern.settings.createDefaultFrontendContext
 
 object LetsPlot {
     var frontendContext: FrontendContext = createDefaultFrontendContext()
 
-    var theme: OptionsMap? by GlobalSettings::theme
+    var theme: Any? = null
```

Comment:
Let's have type `Feature` instead of `Any`.

Add `set()`:
- check 'kind', fail if not `Option.Plot.THEME`
- if `FeatureList`, replace with one `OptionsMap` by merging. Likely reuse `ThemeOptionsUtil.toSpec()`